src/main/java/teammates/ui/controller/AdminSessionsPageAction.java

```diff
@@ -75,15 +77,15 @@ private void putIntoUnknownList(
         }
     }
 
-    private void prepareDefaultPageData(Calendar calStart, Calendar calEnd) {
+    private void prepareDefaultPageData(Instant start, Instant end) {
         this.map = new HashMap<>();
         this.totalOngoingSessions = 0;
         this.totalOpenStatusSessions = 0;
         this.totalClosedStatusSessions = 0;
         this.totalOpenStatusSessions = 0;
         this.totalInstitutes = 0;
-        this.rangeStart = calStart.getTime();
-        this.rangeEnd = calEnd.getTime();
+        this.rangeStart = start;
+        this.rangeEnd = end;
     }
 
     private ActionResult createShowPageResultIfParametersInvalid() {
```

Comment:
I think this name is really misleading when reading `execute`. It's different from other `createShowPage...` methods because it's also extracting the parameters, but the name doesn't reflect that.